governance/polls/Activate Lido Rewards Program - January 8, 2024.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Activate Lido Rewards Program - January 08, 2024
```

Comment:
```suggestion
title: SparkLend Ethereum - Activate Lido Rewards Program - January 08, 2024
```

The titles of all the polls do not clearly indicate that they pertain to Spark. Additionally, they should specify that the intended impact is solely on the Ethereum chain, as Spark also operates on the Gnosis chain. To address this, I recommend adding "SparkLend Ethereum -" at the beginning of each title. It would also be advisable to rename the files accordingly